components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java

```diff
@@ -41,5 +41,5 @@
     public static final String EMAIL_TEMPLATE_TYPE_ACC_DISABLED = "accountdisable";
     public static final String EMAIL_TEMPLATE_TYPE_ACC_ENABLED = "accountenable";
 
-
+    public static final String ACCOUNT_LOCK_BYPASS_ROLE = "Internal/system";
```

Comment:
Check whether we already have a constant for "Internal" since this is a system reserved word. If we have one, things will be easier and cleaner if we use it to prepend.